spec/modules.md

```diff
@@ -262,7 +264,11 @@ This algorithm takes a string `argument` and [configuration](#configuration)
 
   [executed]: spec.md#executing-a-file
 
-  * If `config` is not empty, throw an error.
+  * If `config` has a different ID than before and is not empty, throw an error.
```

Comment:
I think "than the configuration that was passed the first time `file` was executed" would be clearer.

Also, this is a pre-existing issue, but you might as well fix it while you're here: I think the previous bullet point should say "If `file` has already been executed **by the Loading a Module procedure**", since it's valid to load a file with `@import` and also with `@use`.